docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md

```diff
@@ -586,7 +588,30 @@ Make sure that when you add merchant product to cart, on a cart page is has the
 
 {% endinfo_block %}
 
-### 2) Set up behavior
+### 2) Add Yves translations
+
+Append glossary according to your configuration:
+
+**src/data/import/common/common/glossary.csv**
+
+```
+merchant_product.message.invalid,Product "%sku%" with Merchant "%merchant_reference%" not found.,en_US
+merchant_product.message.invalid,Der Produkt "%sku%" mit dem Händler "%merchant_reference%" ist nicht gefunden.,de_DE
```

Comment:
please add other translations related to this feature (check glossary.csv file, for example merchant_product.sold_by)

Reply:
I have added it some where already, but ok. Added.